bson-scala/src/main/scala/org/mongodb/scala/bson/codecs/macrocodecs/CaseClassCodec.scala

```diff
@@ -127,18 +127,15 @@ private[codecs] object CaseClassCodec {
 
     val fields: Map[Type, List[(TermName, Type)]] = {
       knownTypes
-        .map(
```

Comment:
I think this change is okay, so I didnt look to much in finding an option to revert this change